### ISSUE TITLE
Web page news is out of date about versions

### ISSUE THREAD

**Issue by gdt** [cmusatyalab/coda#106]
I noticed on repology.org that pkgsrc was out of date, and the version listed as current does not show up at http://coda.cs.cmu.edu/news.html but yet there is a tag  on github (but not a release).

(Probably I have fallen off the list by entropy, or  maybe I unsubscribed, but in any case I have not tried to run coda lately.  I'll see about updating the pkgsrc package.)

**Comment by thesamesam** [comment]
Right, I see newer tags but no releases and they're not on the FTP site.

**Comment by gdt** [comment]
Thanks, would be nice to update the website.  Also it is http only and that gets a (somewhat valid) Big Scary Warning.

Great to hear about TLS tunnels.

**Comment by thesamesam** [comment]
Thanks for the update. Is rpc2 always just in the coda tarball now? It's not distributed separately?
